apicast/src/configuration.lua

```diff
@@ -171,6 +173,9 @@ function _M.parse_service(service)
         local usage_t =  {}
         local matched_rules = {}
 
+        -- XXX: get_auth_params gets the params from the body for the not-GET calls, which probably 
```

Comment:
It's actually unclear if this is a desired behavior or not. Issue created: https://github.com/3scale/apicast/issues/163